WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+/// Hosting controller for `ProductDescriptionGenerationCelebrationView`.
+///
+final class ProductDescriptionGenerationCelebrationHostingController: UIHostingController<ProductDescriptionGenerationCelebrationView> {
+    init(viewModel: ProductDescriptionGenerationCelebrationViewModel) {
+        super.init(rootView: ProductDescriptionGenerationCelebrationView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Celebration view presented when AI generation is used for the first time
+struct ProductDescriptionGenerationCelebrationView: View {
+    private var viewModel: ProductDescriptionGenerationCelebrationViewModel
```

Comment:
Nit: if this view model is not an `@ObservedObject`, we can keep it a constant instead.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/9939/commits/7802a68f902d3d596430f8cc0dba6e351524518f